azext_iot/tests/digitaltwins/test_dt_resource_lifecycle_int.py

```diff
@@ -425,17 +423,25 @@ def test_dt_endpoints_routes(self):
         for endpoint_name in endpoint_names:
             logger.debug("Cleaning up {} endpoint...".format(endpoint_name))
             is_last = endpoint_name == endpoint_names[-1]
-            delete_ep_output = self.cmd(
+            self.cmd(
                 "dt endpoint delete -n {} --en {} {}".format(
                     endpoints_instance_name,
                     endpoint_name,
                     "-g {}".format(self.dt_resource_group) if is_last else "",
                 )
-            ).get_output_in_json()
-            assert delete_ep_output["provisioningState"] == "Deleting"
-            assert delete_ep_output["id"].endswith("/{}".format(endpoint_name))
-            sleep(15)  # Wait for service to catch-up. Service will fix at some point.
+            )
+            # DT endpoint delete does not currently return data
+            # ).get_output_in_json()
+            # assert delete_ep_output["properties"]["provisioningState"] == "Deleting"
+            # assert delete_ep_output["id"].endswith("/{}".format(endpoint_name))
+            sleep(10)  # Wait for service to catch-up. Service will fix at some point.
```

Comment:
FYI - Apparently there's a service update currently rolling out that should fix this, and endpoint delete (like dt delete) should in fact return an instance of the endpoint.

Reply:
Update - I misunderstood, endpoints (like DT instances) will finish their polling on DELETE with a 204: NoContent